components/brave_perf_predictor/browser/third_party_extractor.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_THIRD_PARTY_EXTRACTOR_H_
+#define BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_THIRD_PARTY_EXTRACTOR_H_
+
+#include <string>
+#include <unordered_map>
+
+#include "base/values.h"
+#include "base/memory/singleton.h"
+
+namespace base {
```

Comment:
you have the header and the forward declaration here. You only need the header and `base` is the one thing you generally don't need to use forward declarations for